man/binom_lik.Rd

```diff
@@ -4,12 +4,15 @@
 \alias{binom_lik}
 \title{Likelihood object for Binomial error distribution}
 \usage{
-binom_lik(y, n)
+binom_lik(y, n, link = c("identity", "logit"))
 }
 \arguments{
 \item{y}{Binomial observations}
 
 \item{n}{Binomial number of trials}
+
+\item{link}{Link function. The "identity" link directly puts unimodal prior on Binomial success
```

Comment:
The posterior is still on p for binomial likelihood with logit link, and the posterior is still on lambda for Poisson likelihood with log link. Now they are clarified in pois_lik.Rd and binom_lik.Rd.